openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -67,16 +101,13 @@ def test_set_credit_requirements_invalid_course(self):
             }
         ]
         with self.assertRaises(InvalidCreditCourse):
-            set_credit_requirements(self.course_key, requirements)
+            api.set_credit_requirements(self.course_key, requirements)
 
         self.add_credit_course(enabled=False)
         with self.assertRaises(InvalidCreditCourse):
-            set_credit_requirements(self.course_key, requirements)
+            api.set_credit_requirements(self.course_key, requirements)
 
     def test_set_get_credit_requirements(self):
-        """Test that if same requirement is added multiple times
```

Comment:
Nose has this irritating habit of using the docstring as the test name in its output, which can make it difficult to find the method corresponding to a test failure.

I've added these docstrings back in as comments; I've also updated some of the new tests with comments describing what they're testing.